src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -653,6 +653,10 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
         $content = $dumper->dump(array('class' => $class, 'base_class' => $baseClass, 'file' => (string) $cache));
         if (!$this->debug) {
             $content = static::stripComments($content);
+            // reclaim memory for php7, as memory manager will not release after token_get_all()
+            if (function_exists('gc_mem_caches')) {
+                gc_mem_caches();
+            }
```

Comment:
If we call `gc_mem_caches()` because of the usage of `token_get_all()`I'd we should move these lines to `stripComments()`.